gizmo.py

```diff
@@ -0,0 +1,29 @@
+# file for python testing test
```

Comment:
Solutions should not be merged into the master branch at this time.

Reply:
Sorry, I merged my solution branch to my test dev branch (stupid idea in hindsight), and didn't manage to not commit the solution files. So, pushing this file was not intentional, sorry it happened.